browser/resources/settings/brave_overrides/privacy_page.ts

```diff
@@ -14,11 +14,12 @@ function InsertGoogleSignInSubpage (
   templateContent: DocumentFragment,
   pages: Element)
 {
-  pages.insertAdjacentHTML(
-    'beforeend',
-    getTrustedHTML`
+  pages.appendChild(
+    html`
```

Comment:
It's surprising me that we don't need to use TrustedHTML for these. They used to spit out a warning and not render properly, from what I recall. If these don't have to be TrustedHTML, there are probably other simplifications we can make to this file, e.g., where we dynamically change the HTML via `setAttribute`, etc. though that can be a follow up I'm sure.

Reply:
I'm pretty sure it's because Google updated Polymer to use TrustedTypes internally - Polymer has it's own TrustedType for HTML is whitelisted in the default WebUI CSP.

Polymer TrustedType: https://github.dev/chromium/chromium/blob/8708e0012a2ccd1e162b9f9b0b852a229df7049c/third_party/polymer/v3_0/components-chromium/polymer/polymer_bundled.js#L8209

CSP: https://github.dev/chromium/chromium/blob/8708e0012a2ccd1e162b9f9b0b852a229df7049c/chrome/browser/ui/webui/webui_util.cc#L82

This is a fairly recent change (~10 months), so I imagine that's what stopped the warnings.